blockmanager.go

```diff
@@ -1223,7 +1259,7 @@ func (b *blockManager) handleInvMsg(imsg *invMsg) {
 			}
 
 			// Add it to the request queue.
-			requestQueue = append(requestQueue, iv)
+			state.requestQueue = append(state.requestQueue, iv)
```

Comment:
This is incorrect.  It should be using the local request queue, not a persistent per-peer request queue.